### ISSUE TITLE
Copy Past Filter salary

### ISSUE THREAD

**Issue by JuanVaquiro** [midudev/sueldos.dev#13]
when I selected the salary I wanted to convert to my currency. The copy turned out to be a very long code.

![image](https://user-images.githubusercontent.com/64930246/230456414-65955d61-ee3d-40f7-a4d2-2aeced1be1e5.png)

**Comment by wilsenhc** [comment]
I did some research on this and it seems to be due to how the `react-animated-numbers` works internally. Component is not good on a11y at all.

A solution could be to replace this package with another one or build a custom component for that or just use a normal CSS transition when updating the salary.

**Comment by tiborux** [comment]
Another easy solution will be create a button 'wrapping' the component and allows you to copy the value directly:

```
<button @click="copyToClipboard(result.salary)">
    <AnimatedNumbers locale='es-ES' includeComma animateToNumber={result.salary} />€
 </button>
```

It is not the best solution, but I could work for the moment.

I was also thinking to create a span showing the salary when the component finish its animation, but the component doesnt export a way to know when finish its animation.

**Comment by voluspak** [comment]
Resolved the issue in Pull Request #36 👍🏻 😃
